cmd/osv-scanner/main.go

```diff
@@ -128,6 +128,11 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Name:  "experimental-licenses",
 				Usage: "report on licenses",
 			},
+			&cli.BoolFlag{
+				Name:  "disable-call-analysis",
```

Comment:
@another-rex @G-Rath we discussed making this "no-call-analysis" to be consistent with "no-ignore". 

@hogo6002 could you please make this change?

Reply:
Sure, just updated.